libs/common/src/auth/services/key-connector.service.ts

```diff
@@ -16,24 +24,47 @@ import { KeyConnectorUserKeyRequest } from "../models/request/key-connector-user
 import { SetKeyConnectorKeyRequest } from "../models/request/set-key-connector-key.request";
 import { IdentityTokenResponse } from "../models/response/identity-token.response";
 
+export const USES_KEY_CONNECTOR = new KeyDefinition<boolean>(
+  KEY_CONNECTOR_DISK,
+  "usesKeyConnector",
+  {
+    deserializer: (usesKeyConnector) => usesKeyConnector,
+  },
+);
+
+export const CONVERT_ACCOUNT_TO_KEY_CONNECTOR = new KeyDefinition<boolean>(
+  KEY_CONNECTOR_DISK,
+  "convertAccountToKeyConnector",
+  {
+    deserializer: (convertAccountToKeyConnector) => convertAccountToKeyConnector,
+  },
+);
```

Comment:
If you want to take @jlf0dev 's [suggestion](https://github.com/bitwarden/clients/pull/8327/files#r1526486161):

```suggestion
export const CONVERT_ACCOUNT_TO_KEY_CONNECTOR = new UserKeyDefinition<boolean>(
  KEY_CONNECTOR_DISK,
  "convertAccountToKeyConnector",
  {
    deserializer: (convertAccountToKeyConnector) => convertAccountToKeyConnector,
    clearOn: ["logout"],
  },
);
```

import `UserKeyDefinition` from the same location as `KeyDefinition`

Reply:
I was trying to fund usage examples, thanks!